content/en/agent/guide/upgrade-to-agent-v6.md

```diff
@@ -19,23 +19,23 @@ Agent 6 is the latest major version of the Datadog Agent. The big difference bet
   * It uses fewer file descriptors
   * It has an all around decreased footprint
 
-* Agent 6 uses ports `5000` and `5001` by default. You can specify different ports for `expvar_port` and `cmd_port` in the `datadog.yaml` file.
+* Agent 6 uses ports `5000` to expose its runtime metrics and `5001` for [its CLI/GUI commands][1]. You can specify different ports for `expvar_port` and `cmd_port` in the `datadog.yaml` file.
```

Comment:
we may want to write that these are _additional_ ports, to make it obvious that for instance the dogstatsd port (`8125`) isn't affected by this sentence